vivisect/analysis/__init__.py

```diff
@@ -105,11 +105,11 @@ def addAnalysisModules(vw):
             vw.addFuncAnalysisModule('vivisect.analysis.arm.thunk_reg')
             vw.addFuncAnalysisModule('vivisect.analysis.arm.renaming')
 
-        vw.addAnalysisModule("vivisect.analysis.generic.funcentries")
         vw.addAnalysisModule("vivisect.analysis.generic.relocations")
         vw.addAnalysisModule("vivisect.analysis.elf.libc_start_main")
-        vw.addAnalysisModule("vivisect.analysis.generic.pointertables")
         vw.addAnalysisModule("vivisect.analysis.generic.emucode")
+        vw.addAnalysisModule("vivisect.analysis.generic.pointertables")
+        vw.addAnalysisModule("vivisect.analysis.generic.funcentries")
```

Comment:
i'm not sure which would be better before which, but i'm guessing (very small opinion) you currently have them ordered the best.  but i certainly like having all pointer analysis after the other analysis modules.

Reply:
yea, keeping things specific -> general is nice. 

With that in mind, the more I think about it, the more I think funcentries should come before pointertables since it is more specific about the type.